doc/nvim-tree-lua.txt

```diff
@@ -106,6 +106,8 @@ function.
         height = 30,
         side = 'left',
         auto_resize = false,
+	number = false,
+	relativenumber = false,
```

Comment:
wrong indent ^^' i believe i use spaces to indent help files.

Reply:
Fixed! I also change another indentation with the same problem just below mine :mage_man: 

before:

![before](https://user-images.githubusercontent.com/32601980/143766464-d30004c5-8284-49eb-819e-160a904985fa.png)

after:

![after](https://user-images.githubusercontent.com/32601980/143766467-4b201679-f4ad-430e-8deb-fed81a690904.png)